services/dr-manhattan/louvaine.py

```diff
@@ -70,12 +72,12 @@ def get_text_sum(self, cluster, r_o):
                     else:
                         r_o["campaigns"]["ids"][cam] = 1
 
-            if r is not None:
-                for res in r.json():
+            if entities:
+                for res in entities:
```

Comment:
assume u mean `json()` bits. its going thru new Loopy helpers so erthing is json object.